Resources/Computing/ComputingElement.py

```diff
@@ -40,10 +40,12 @@
 from DIRAC.Core.Utilities.Time                        import dateTime, second
 from DIRAC                                            import S_OK, S_ERROR, gLogger, version
 from DIRAC.Core.Utilities.ObjectLoader                import ObjectLoader
+from DIRAC.Core.Utilities.Os                          import getNumberOfCores
```

Comment:
Yes, true. I guess this method used to be more involved in the past. Will remove it.